pkg/didcomm/packer/legacy/authcrypt/unpack.go

```diff
@@ -147,6 +160,9 @@ func decodeSender(b64Sender string, pk []byte, km legacykms.KeyManager) ([]byte,
 	senderData := base58.Decode(string(senderPub))
 
 	senderPubCurve, err := cryptoutil.PublicEd25519toCurve25519(senderData)
+	if err != nil {
+		return nil, nil, err
```

Comment:
please wrap error

Reply:
For security reasons, we should not provide hints on any crypto related logic like key conversions, decoding ciphertext, iv, tag, encrypting or decrypting in general, etc.

Reply:
done